repos/system_upgrade/el7toel8/actors/migratentp/tests/unit_test.py

```diff
@@ -0,0 +1,95 @@
+from leapp.libraries.actor import library
+from leapp.libraries.common import reporting
+
+class report_generic_mocked(object):
```

Comment:
I can rename the classes, but I've copied this from other tests that are already included in leapp-repository, so it would be nice to have some consistency here.